week3_model_free/qlearning.ipynb

```diff
@@ -126,7 +126,7 @@
     "\n",
     "    def get_best_action(self, state):\n",
     "        \"\"\"\n",
-    "        Compute the best action to take in a state (using current q-values). \n",
+    "        Compute the best action to take in the state (using current q-values). \n",
```

Comment:
I think "in a state" is fine here. Consider a similar phrase, where, I think, it is clear that "a" should be used:

> For _a_ given state, we shall compute the best action using current Q-values.